torch/distributed/elastic/rendezvous/dynamic_rendezvous.py

```diff
@@ -1198,11 +1198,7 @@ def next_rendezvous(self) -> RendezvousInfo:
             # To avoid race in get_free_port because we release the port after the call,
             # we want to create a TCPStore server soon afterwards.
             server_port = 0
-            addr = (
-                self._store.host
-                if isinstance(self._store, dist.TCPStore)
-                else self._this_node.addr
-            )
+            addr = self._this_node.addr
```

Comment:
Can we hold on a little bit. Originally the change was like this, but it got reverted because some internal test failed. So we need to investigate that.